angular/e2e/feedback/feedback.e2e-spec.ts

```diff
@@ -9,18 +17,25 @@ describe('Feedback', () => {
     page = new FeedbackPage();
   });
 
+  for(let i=0; i<csvValues.data.length-1; i++) {
+    let j = 0;
   // Check if Feedback-Page is loaded
   it('should load Home-Page ', () => {
     page.navigateToFeedback();
     expect(page.getParagraphText()).toEqual('Feedback');
   });
 
   // Check if textfield appears after star-rating, submit and check if it is approved
-  it('feedback page opens and elements are dynamically', () => {
-    page.navigateToFeedback();
-    page.setStarRating();
-    expect(element(page.getTextField).isDisplayed()).toBe(true);
-    page.submitFeedback();
-    expect(element(page.getThanks).isDisplayed()).toBe(true);
-  });
+    it('feedback page opens and elements are dynamically', () => {
+      page.navigateToFeedback();
+      expect(page.getParagraphText()).toEqual('Feedback');
+      page.setStarRating();
+      expect(element(page.getTextField).isDisplayed()).toBe(true);
+      element.all(by.css('.star')).get(csvValues.data[i][j] - 1).click(); //int
```

Comment:
Keine locator innerhalb der Testschritte verwenden, dadurch werden die Tests schwer wartbar, wenn sich was an einer css  Klasse ändert.
- [ ]  Auslagern in Page Objekt
Am besten gleich als funktion (z.B.: setRating(score: int))